get-message-service-bus-queue-peek-lock-renew-sequential-convoy/manifest.json

```diff
@@ -0,0 +1,69 @@
+{
+    "title": "Azure Service Bus: Get message from a queue with token renewal - Peek lock renew pattern",

```

Comment:
```suggestion
    "title": "Azure Service Bus (sequential convoy): Get message from a queue with token renewal - Peek lock renew pattern",

```

Reply:
This version could work too: 

"Azure Service Bus: Get message from a queue with token renewal - Peek lock renew pattern (sequential convoy)"